tests/DCPS/ContentFilteredTopic/ContentFilteredTopicTest.cpp

```diff
@@ -149,10 +149,26 @@ bool run_filtering_test(const DomainParticipant_var& dp,
     return false;
   }
 
+  // Try to set expression parameters that have a parameter too much
+  // compared to the expression, this should fail
+  DDS::StringSeq params_invalid(2);
+  params_invalid.length(2);
+  params_invalid[0] = "2";
+  params_invalid[1] = "2";
+  if (cft2->set_expression_parameters(params_invalid) != RETCODE_ERROR) {
+    cout << "ERROR: setting too much parameters should return an error"
+         << endl;
+    return false;
+  }
+
```

Comment:
would be good to also test too few parameters (more serious error), and also passing these sequences to DomainParticipant::create_contentfilteredtopic()